kafka-spring-boot-autoconfigure/src/test/java/org/axonframework/extensions/kafka/SubscribingProducerIntegrationTest.java

```diff
@@ -47,7 +47,7 @@ public void shouldPublishMessagesSynchronously() throws Exception {
         eventBus.publish(new GenericEventMessage<>("test"));
 
         // then
-        assertThat(Thread.currentThread().getId()).isEqualTo(threadIdCaptor.getThreadId());
```

Comment:
I believe we can remove the `public` modifier from this method and from the test class. Or is this not an option because it's a `@SpringBootTest`?